apps/mobile/src/features/Settings/SettingsScreenNavigator.tsx

```diff
@@ -49,7 +49,7 @@ const Stack = createStackNavigator<SettingsStackRoutes>()
 export const SettingsScreenNavigator = () => {
   const strings = useStrings()
   const {track} = useMetrics()
-  const {atoms, color} = useTheme()
+  const {atoms, palette} = useTheme()
```

Comment:
stick to conventions
```suggestion
  const {atoms, palette: p} = useTheme()
```